index.html

```diff
@@ -0,0 +1,469 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>Youtube Page</title>
+  </head>
+
+  <body>
+    <nav class="flex">
+      <div id="left-navbar" class="flex">
+        <a href="#"><img src="assets/hamburger.png" alt="" /></a>
+        <a href="#"><img src="assets/youtubelogos.png" alt="" /></a>
+      </div>
+      <div id="mid-navbar" class="flex">
+        <input id="search-bar" type="text" placeholder="Search" />
+        <button id="search-button">
+          <i class="fa fa-search" aria-hidden="true"></i>
+        </button>
+      </div>
+
+      <div id="right-navbar" class="flex">
+        <a href="#"><img src="assets/nav1.png" alt="" /></a>
+        <a href="#"><img src="assets/nav2.png" alt="" /></a>
+        <a href="#"><img src="assets/nav4.png" alt="" /></a>
+        <a href="#"><img src="assets/nav3.png" alt="" /></a>
+      </div>
+    </nav>
+
+    <div class="d-flex space-between">
+      <div class="main-fc d-flex flex-column">
+        <div class="container">
+          <div class="main-video">
+            <iframe
+              width="853"
+              height="480"
+              src="https://www.youtube.com/embed/YbYWhdLO43Q"
+              allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+            <h3 class="video-title">
+              This Unicorn Changed the Way I Poop -
+              <a
+                href="https://www.youtube.com/results?search_query=%23SquattyPotty"
+                >#SquattyPotty</a
+              >
+            </h3>
+            <div id="video-below" class="flex flex-horizontal">
+              <div class="video-views">
+                <p class="smaller-gray">38,063,145 views • Oct 6 2015</p>
+              </div>
+              <div class="flex">
+                <i class="fa fa-thumbs-up icon-spacing" aria-hidden="true">
+                  115k</i
+                >
+
+                <i class="fa fa-thumbs-down icon-spacing" aria-hidden="true">
+                  5.8K</i
+                >
+                <i class="fa fa-share icon-spacing"> SHARE</i>
```

Comment:
You need to make this a button. It is a requirement. You can wrap it using`<button>` tag